src/plugins/intel_gpu/src/plugin/ops/rnn.cpp

```diff
@@ -64,6 +66,58 @@ void GetLSTMActivationParams(const std::shared_ptr<T>& op,
     }
 }
 
+template <typename T>
+void GetGRUActivationParams(const std::shared_ptr<T>& op,
+                             std::vector<cldnn::activation_func>& activations,
+                             std::vector<cldnn::activation_additional_params>& activation_params) {
+    activations = { cldnn::activation_func::logistic,
+                    cldnn::activation_func::hyperbolic_tan };
+    activation_params = {};
+    auto op_activations = op->get_activations();
+    if (!op_activations.empty()) {
+        if (op_activations.size() != 2)
+            OPENVINO_THROW("Wrong number of activations for GRUSeq op ", op->get_friendly_name());
+        for (int i = 0; i < 2; i++) {
+            auto af = GetActivationFunc(op_activations[i]);
+            if (af == cldnn::activation_func::none)
+                OPENVINO_THROW("Wrong or unsupported activation type ", op_activations[i], " for GRUSeq op ", op->get_friendly_name());
+            activations[i] = af;
+        }
+    }
+    auto op_a = op->get_activations_alpha();
+    auto op_b = op->get_activations_beta();
+    if (!op_a.empty()) {
+        if (op_a.size() != 2 || op_b.size() != 2)
```

Comment:
nit: It is better to use `OPENVINO_ASSERT` here, several similar cases in this file